documentation/src/components/code-example.css

```diff
@@ -19,13 +19,25 @@ governing permissions and limitations under the License.
     width: 100%;
 }
 
-#demo {
+.demo-example {
     flex: 1;
     padding: 3rem 4rem;
     border-radius: 6px 6px 0 0;
     background: var(--spectrum-global-color-gray-100);
 }
 
+@media (max-width: 768px) {
+    .demo-example {
+        padding: 2.25rem 3rem;
+    }
+}
+
+@media (max-width: 568px) {
+    .demo-example {
+        padding: 1.5rem 2rem;
+    }
+}
```

Comment:
768 is iPad, so that seems like a good breakpoint (though maybe I should go `767` for the changes?).

568 is just 200 less than that. No real reason, but it seems like a good point to drop the padding down further.

Reply:
I might as well to switch to those dimension variables (but try to bottom our a little lower?)

I think we'll need something different for the `sp-theme` examples where the page itself controls the sizing since it's so tight. Don't want to tackle that here, outside of getting the right followup issue written. Do you think something like `html demo-no-padding` or `html demo-custom` would be the right name? (I haven't quite figured out how that doc section actually gets mapped to the `code-example` files btw).

Reply:
That's tricky. I've been working on track reimagining of the docs site, but without any specific reason to publish it hadn't prioritized getting it into a PR. If you're going to be doing a lot more docs work, would it be useful for me to work it into PR (even if just into a feature branch) so that we can work from there. Addressing something like this might fall in line with how it prints the documentation into the light DOM, which should give us some more flexibility in customizing this padding.

Reply:
I wouldn't count on me doing a ton of work near term -- I just wanted to get back to fixing some issues since I'm filing a bunch, and docs were the easiest thing to start with.